### ISSUE TITLE
Porting the splash screen and some other cosmetic stuff to Katana

### ISSUE THREAD

**Issue by RandomGuy38** [fluxer/katana#39]
I really like the work you've done with Katana (I like KDE 4 in general, and it's nice that Plasma and the applications were ported to newer distros/distro versions), though I do have a few (three) feature requests:

Would it be possible to add the Plasma splash screen to Katana?
And can you add the KWin fade effect for when windows appear and dissapear and "fix" the minimize effect? (basically when you minimize a window it doesn't go to the taskbar but it shrinks to the middle of the desktop, like when plasmashell isn't running but you minimize a window)
There is an issue with VSync not working/still lots of screen tearing, (it was also present in KDE 4 and older Plasma 5 versions) I don't know if it's possible to fix that though.

It's all small stuff though, so it's no big deal if you don't add those right away or not at all.

**Comment by fluxer** [comment]
Hi,

Glad you like it!

It is possible to add splash screen, in fact there was one. Practically while the splash screen would be running interaction may
be required such entering password for the shared directories, rather than just hide that behind something I prefer to get it out of the way as soon as possible. Otherwise the desktop may be playing notification sounds and such without knowing what is
actually going behind that splash screen so it was removed and I wont have it back.

KWin has a minimize animation effect and the same effect can probably be used for when closing windows and when windows are shown for the first time but that is extra eye-candy that I am not in for it right now. At some point I will probably add more effects altought I am not sure how much that will be worth considering that there is, for example, [compiz](https://en.wikipedia.org/wiki/Compiz) with lots of eye-candy but is unmaintained (not much of recent activity). While [it is supported](https://github.com/fluxer/kde-workspace/blob/master/kcontrol/componentchooser/windowmanagers/compiz.desktop) it may require configuration to get compiz to work but if you are into eye-candy you may want to look into it.

As for "fixing" the minimize effect I am guessing that is how it is supposed to work but I agree it is kinda odd. On the other side KWin can only guess where the position of the task (it's visual representation) on the taskbar is - the panel can be on top, bottom, right, left. Heck, minimization can happen programatically without even a taskbar being there (no panel at all). There are a lot of little things that have to be taken into account when "fixing" that and I believe it works as it should - without too many  things to go wrong. The effect can always be disabled.

Synchronization is always an issue, resizing and scrolling is something other projects have issues with too (even if not admitted). The more the processing power the less of an issue it is usually is.

Long story short - works as intended, except any synchronization issues but those may be even caused by X11, the video driver for X11 or even a kernel scheduling doing something funky. It is a rabbit hole by itself.

Cheers!

**Comment by fluxer** [comment]
Closing due to the questions being answered.
